src/main/webapp/WEB-INF/datastore-indexes.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<datastore-indexes autoGenerate="false">
```

Comment:
Yea I just thought of that it's better to make it fail. Hence editing my comment to have a comment on it. Didn't know you can saw it LOL.